exec/vector/src/main/java/org/apache/drill/exec/vector/accessor/convert/AbstractConvertFromString.java

```diff
@@ -26,9 +31,214 @@
  */
 public abstract class AbstractConvertFromString extends AbstractWriteConverter {
 
+  /**
+   * Property to control how the conversion handles blanks. Blanks are
+   * zero-length text fields (after triming whitespace.)
+   */
+  public static final String BLANK_ACTION_PROP = "blank-as";
```

Comment:
I did consider an enum. However, since these are just strings, the extra mechanism of an enum did not seem to add value: we would never actually use the enum.